public/app/features/dashboard/components/TransformationsEditor/TransformationsEditor.tsx

```diff
@@ -45,11 +53,22 @@ interface TransformationsEditorProps extends Themeable {
   panel: PanelModel;
 }
 
+type FilterCategory = TransformerCategory | 'viewAll';
+
+const filterCategoriesLabels: Array<[FilterCategory, string]> = [
+  ['viewAll', 'View all'],
+  ...(Object.entries(categoriesLabels) as Array<[FilterCategory, string]>),
```

Comment:
I don't love the type assertion but I don't see a way around it without removing the enum.

Reply:
I agree